test/houghtransform.jl

```diff
@@ -36,4 +36,30 @@ using ImageFeatures
         er = sum(map((t1,t2) -> abs(t1-t2), theta, [0, π/2, π/2, 0]))
         @test er <= 0.1
     end
+
+    @testset "Hough Circle Gradient" begin
+
+    dist(a, b) = sqrt(sum(abs2, (a-b).I))
+
+    img=zeros(Integer, 300, 300)
+    for i in CartesianRange(size(img))
+        if dist(i, CartesianIndex(100, 100))<25 || dist(i, CartesianIndex(200, 200))<50
+            img[i]=1
+        else
+            img[i]=0
+        end
+    end
+
+    img_edges = canny(img, 1, 0.2, 0.1, percentile=false)
```

Comment:
Obviously this will need to be updated once a new Images gets tagged, but that needs to wait on https://github.com/JuliaImages/Images.jl/pull/617.